packages/server/modules/shared/helpers/envHelper.ts

```diff
@@ -81,6 +81,22 @@ export function getOidcName() {
   return process.env.OIDC_NAME
 }
 
+export function getMailchimpConfig() {
+  if (
+    !process.env.MAILCHIMP_API_KEY ||
```

Comment:
How will this work locally? Should this just be disabled or should we set up actual mailchimp lists for dev purposes?

Reply:
added an extra env var, we can ignore it locally